ftw/solr/helpers.py

```diff
@@ -1,7 +1,27 @@
+from functools import partial
 from itertools import izip
 
 
+MAXCHUNKSIZE = 1 << 16
```

Comment:
A chunk size of 64K looks pretty small to me.
I din't find any recommendations but in terms of memory consumption 1 MB wouldn't be an issue.
However I'm not sure about the performance impact.

Reply:
This is based on the [`MAXCHUNKSIZE` in `opengever.base.namedfile`](https://github.com/4teamwork/opengever.core/commit/18d9f3faf3028d9d67af7abce855e0528cc497d6), which in turn seems to be derived from the likes of [`plone.namedfile`](https://github.com/plone/plone.namedfile/blob/master/plone/namedfile/file.py#L29) and [`zope.app.file`](https://github.com/zopefoundation/zope.app.file/blob/master/src/zope/app/file/file.py#L25).

Usage is not exactly the same (reading from local files vs. setting HTTP Chunk size), but I failed to find any recommendations for HTTP chunk size whatsoever.